substrate/frame/revive/src/vm/evm/instructions/contract.rs

```diff
@@ -216,38 +170,34 @@ pub fn delegate_call<'ext, E: Ext>(context: Context<'_, 'ext, E>) {
 	popn!([local_gas_limit, to], context.interpreter);
 	let to = Address::from_word(B256::from(to));
 	// Max gas limit is not possible in real ethereum situation.
-	let local_gas_limit = u64::try_from(local_gas_limit).unwrap_or(u64::MAX);
+	let _local_gas_limit = u64::try_from(local_gas_limit).unwrap_or(u64::MAX);
```

Comment:
let's mention the issue where this should be fixed if we can

Reply:
done